draft-ietf-dots-requirements.md

```diff
@@ -245,13 +245,15 @@ for more rapid attack response against future attacks, as all interactions
 setting up DOTS, including any business or service level agreements, are already
 complete.
 
+Multi-homed DOTS clients must maintain multiple DOTS associations with multiple upstream DOTS servers.   
```

Comment:
This seems out of place here. I was thinking of moving this and the other change to a new paragraph, and rewording to something like:
```A DOTS deployment may be multi-homed. DOTS clients in such deployments maintain associations with multiple DOTS servers. Multi-homed DOTS clients must be able to select ....```.